Makefile

```diff
@@ -81,27 +82,42 @@ vet:
 # Run go lint.
 lint:
 	golint || go get -u golang.org/x/lint/golint
-	golint -set_exit_status ./cmd/... ./pkg/... ./clusterctl/...
+	golint -set_exit_status ./cmd/... ./pkg/...
 
 # Build the docker image
 docker-build: generate fmt vet
-	docker build . -t ${IMG}
+	docker build . -t ${MANAGER_IMAGE}
 
 # Push the docker image
 docker-push:
-	docker push ${IMG}
+	docker push ${MANAGER_IMAGE}
 
 # Cleanup
 clean:
-	rm -rf out
+	rm -rf cmd/clusterctl/examples/aws/out/
```

Comment:
I think we should consider keeping the generated yaml closer to the root of the repo to avoid the user having to reference deep nested paths when using the generated files.